src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,191 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private int size;
+    private Node<T> head;
+    private Node<T> tail;
+
+    public static class Node<T> {
```

Comment:
```suggestion
    private static class Node<T> {
```

as we use this class only inside MyLinkedList.class